File: Ejercicios/Java/Ejercicio5.java
import java.util.ArrayList;
class Alumno {
    String nombre; double nota;
    Alumno(String n, double p) { nombre = n; nota = p; }
}
public class Ejercicio5 {
    public static void main(String[] args) {
        ArrayList<Alumno> clase = new ArrayList<>();
        clase.add(new Alumno("Victor", 9.5));
        clase.add(new Alumno("Maria", 8.0));
        for(Alumno a : clase) System.out.println(a.nombre + " - " + a.nota);
    }
}

File: Ejercicios/C++/Ejercicio5.cpp
#include <iostream>
#include <vector>
using namespace std;
class Alumno {
public: string nombre; double nota;
    Alumno(string n, double p) : nombre(n), nota(p) {}
};
int main() {
    vector<Alumno> clase;
    clase.push_back(Alumno("Victor", 9.5));
    clase.push_back(Alumno("Maria", 8.0));
    for(Alumno a : clase) cout << a.nombre << " - " << a.nota << "\n";
    return 0;
}
